gef.py

```diff
@@ -1283,11 +1283,9 @@ def get_heap_for_ptr(ptr):
         return ptr & ~(heap_max_size - 1)
 
     def __str__(self):
-        fmt = "{:s}(base={:#x}, top={:#x}, last_remainder={:#x}, next={:#x}, next_free={:#x}, system_mem={:#x})"
-        return fmt.format(
-            Color.colorify("Arena", "blue bold underline"),
-            self.__addr, self.top, self.last_remainder, self.n, self.nfree, self.sysmem
-        )
+        return f"""{Color.colorify("Arena", "blue bold underline")}(base={self.__addr:#x}, top={self.top:#x}, """\
```

Comment:
you don't have to change this but fyi you can put this is parens to avoid the `\`.